core/build.gradle

```diff
@@ -55,6 +55,8 @@ protobuf {
 tasks.withType(AbstractArchiveTask) {
     preserveFileTimestamps = false
     reproducibleFileOrder = true
+    dirMode = 0755
```

Comment:
This is interesting – it could perhaps replace the `chmod` I added earlier?

Reply:
However, it will likely remove the `x` bit from the executables – the `X` syntax keeps those.

Reply:
I don't think we have any of executables inside JARs, right?

Reply:
> This is interesting – it could perhaps replace the chmod I added earlier?

Yes. In my testing it works for that purpose.